README.md

```diff
@@ -14,6 +14,10 @@ Please check the [contributing guide](https://www.x-lab.info/github-analysis-rep
 
 Please check the [architect](https://www.x-lab.info/github-analysis-report-2020/) and [workflow](https://www.x-lab.info/github-analysis-report-2020/) if you want to better understand the project.
 
+## Communication
+
+Click here to join the [slack communication group](https://join.slack.com/t/x-github-analysis/shared_invite/zt-gz8ykv3l-qP53IpajsZIGsx6dFo_8Lg) if you want to know more about the project and join it.
```

Comment:
Sorry, but I think `Slack` should be capitalized.

Reply:
And if you use `Click here to join`, maybe the url should be added on `here` but not on `Slack communication group`

Reply:
OK